ui/src/main/resources/org/apache/hop/ui/hopgui/file/workflow/messages/messages_it_IT.properties

```diff
@@ -152,13 +152,13 @@ HopGui.Entries=Entry
 HopGui.Error=Errore
 HopGui.Error.UnableToLoadSharedObjects.Message=Errore durante il caricamento degli oggetti condivisi 
 HopGui.Error.UnableToLoadSharedObjects.Title=Errore
-HopGui.ErrorDialog.ErrorEditingAction.Message=Si � verificato un errore durante la modifica della action
+HopGui.ErrorDialog.ErrorEditingAction.Message=Si � verificato un errore durante la modifica della action
```

Comment:
merge conflict in file

Reply:
My bad. I wasn't thinking I had this issue. I will fix it